pom.xml

```diff
@@ -25,13 +25,13 @@
 
         <glistenVersion>6.0.4</glistenVersion>
         <glistenAfterburnerVersion>2.0.4</glistenAfterburnerVersion>
-        <attachVersion>4.0.8-SNAPSHOT</attachVersion>
-        <mapsVersion>2.0.0-ea+3</mapsVersion>
+        <attachVersion>4.0.11</attachVersion>
+        <mapsVersion>2.0.0-ea+4</mapsVersion>
         <connectVersion>2.0.1</connectVersion>
-        <javafxVersion>15-ea+3</javafxVersion>
-        <controlsfxVersion>11.0.1</controlsfxVersion>
-        <javafx-maven-plugin-version>0.0.4</javafx-maven-plugin-version>
-        <client-maven-plugin-version>0.1.24-SNAPSHOT</client-maven-plugin-version>
+        <javafxVersion>16</javafxVersion>
```

Comment:
Not that it matters that much, but JavaFX static for native-image is already using 17-ea+14.